src/networking.c

```diff
@@ -117,6 +118,65 @@ int authRequired(client *c) {
     return auth_required;
 }
 
+sds getAllClientMetaFields(dict *meta) {
+    dictIterator *di = dictGetIterator(meta);
+    dictEntry *de;
+    sds result = sdscatfmt(sdsempty(),
+        "%%%i\n",
+        (int) dictSize(meta)
+    );
+
+    while((de = dictNext(di)) != NULL) {
+        sds key = dictGetKey(de);
+        sds value = dictGetVal(de);
+
+        result = sdscatfmt(result,
+            "+%s\n:%s\n",
```

Comment:
This is really odd formatting. It looks like the RESPV3 Map response but it's printed out verbatim as a string. I thought in an earlier iteration it looked differently. Something like:
```
meta=key1:value1,key2:value2
```